docs/onboarding_offboarding/Offboarding/admin_checklist.md

```diff
@@ -0,0 +1,49 @@
+# Admin Offboarding Instructions
+
+This checklist is for the lab coordinator running the offboarding process.
```

Comment:
Change lab coordinator to assigned lab staff. In title, I am the lab coordinator, so confusing.